app/util/analytics/analytics_utils.py

```diff
@@ -62,19 +63,20 @@ def generate_report_summary(collector):
 
     summary_report.append(f'Finished|{finished}')
     summary_report.append(f'Compliant|{compliant}')
-    summary_report.append(f'Success|{success}\n')
+    summary_report.append(f'Success|{success}')
+    summary_report.append(f'Has app-specific actions|({bool(collector.app_specific_rates)})')
```

Comment:
Could you pls double check if () are needed.

Reply:
We decided that not needed